cmd/store/import.go

```diff
@@ -183,8 +184,6 @@ func importStore(
 		return nil, fmt.Errorf("failed to finish progress bar: %w", err)
 	}
 
-	fmt.Println("✅ Store imported")
-
```

Comment:
Also, I think it's fine to remove this, because it might mis-lead in some cases where store are getting "created" and not "imported"